docs/extensions/overview.md

```diff
@@ -30,24 +30,24 @@ select * from duckdb_extensions();
-| sqlite_scanner    | Adds support for reading SQLite database files                       | sqlite, sqlite3 |
-| substrait        | Adds support for the Substrait integration                           |
-| tpcds             | Adds TPC-DS data generation and query support                        |                 |
-| tpch              | Adds TPC-H data generation and query support                         |                 |
-| visualizer        |                                                                      |                 |
+| Extension name                                                     | Description                                                          | Aliases         |
+| ------------------------------------------------------------------ | -------------------------------------------------------------------- | --------------- |
+| autocomplete                                                       | Adds supports for autocomplete in the shell                          |                 |
+| [arrow](https://github.com/duckdblabs/arrow)                       | A zero-copy data integration between Apache Arrow and DuckDB         |                 |
+| excel                                                              |                                                                      |                 |
+| fts                                                                | Adds support for Full-Text Search Indexes                            |                 |
+| httpfs                                                             | Adds support for reading and writing files over a HTTP(S) connection | http, https, s3 |
+| icu                                                                | Adds support for time zones and collations using the ICU library     |                 |
+| inet                                                               | Adds support for IP-related data types and functions                 |                 |
+| jemalloc                                                           | Overwrites system allocator with JEMalloc                            |                 |
+| json                                                               | Adds support for JSON operations                                     |                 |
+| parquet                                                            | Adds support for reading and writing parquet files                   |                 |
+| [postgres_scanner](https://github.com/duckdblabs/postgres_scanner) | Adds support for reading from a Postgres database                    | postgres        |
+| [spatial](https://github.com/duckdblabs/duckdb_spatial)            | Adds support for geospatial data processing                          |                 |
+| [sqlite_scanner](https://github.com/duckdblabs/sqlite_scanner)     | Adds support for reading SQLite database files                       | sqlite, sqlite3 |
```

Comment:
Maybe these should be links to the docs instead? And we can add octocats (:octocat:) to link to the repos, as they'll be less often used

Reply:
Good point. I linked to the docs now except for Arrow where we currently only have guides.